### PR TITLE
Add publish step to woodpecker

### PR DESCRIPTION
**Pull request by SleeplessOne1917**

### CODE REVIEW

File: .woodpecker.yml [hunk 1]
```diff
@@ -76,17 +76,16 @@ steps:
...
-  #   secrets: [cargo_api_token]
-  #   when:
-  #     - event: tag
+  publish_to_crates_io:
+    image: *rust_image
+    commands:
+      - cargo install cargo-workspaces
+      - cp -r migrations crates/db_schema/
+      - cargo login "$CARGO_API_TOKEN"
+      - cargo workspaces publish --from-git --allow-dirty --no-verify --allow-branch "${CI_COMMIT_TAG}" --yes custom "${CI_COMMIT_TAG}"
```
Comment (dessalines) [inline .woodpecker.yml:85]: I don't think you need lines 82-84, and probably don't need cargo workspaces either.

**Review by dessalines — COMMENTED** [review]
Make sure to test this by pushing up an alpha tag, something like `x.x.x-alpha.X`

**Comment by SleeplessOne1917** [comment]
It took a bit of tweaking, but I got it to publish successfully.

**Review by dessalines — APPROVED** [review]
Nice, looks good.

**Pull request merged**